packages/console/src/components/MultiTextInput/MultiTextInput.module.scss

```diff
@@ -14,4 +14,10 @@
       margin-right: _.unit(3);
     }
   }
+
+  .error {
+    font: var(--font-body-2);
```

Comment:
Please be noted that `--font-body-2` has been updated to `--font-body-medium` in the latest design. I have a PR to update the fonts in our code base (#462). If #462 gets merged before your PR, please remember to update the fonts used in your code

Reply:
Update: #462 has been merged. So please update this font usage.

Reply:
Will rebase & updated before merge

Reply:
updated

Reply:
updated